recipes/etc-local.rb

```diff
@@ -30,6 +30,11 @@
       config_path = site['docroot']
     end
 
+    directory "#{config_path}/app/etc" do
+      recursive true
+      action :create
+    end
+
```

Comment:
no worries, I wasn't aware of that being the default. Can you rebase -i to clean up some of the adding and removing?

I see you've added the guard, which I'm happy to be removed based on what you confirmed